engine/glfw/lib/cocoa/cocoa_init.m

```diff
@@ -88,33 +88,6 @@ - (void)setAppleMenu:(NSMenu *)m;
         }
     }
 
-    // If we get here, we're unbundled
-    if( !_glfwLibrary.Unbundled )
-    {
-        // Wait for an (any) event before calling SetFrontProcess() below.
-        // If we skip this, sometimes the app/window become unresponsive
-        // right after start.
-        //
-        // This also fixes the bug where the correct menu won't show
-        // unless the user switches from the app and back again.
-        //
-        // Eclipse does something similar:
-        // https://bugs.eclipse.org/bugs/attachment.cgi?id=248969
-        [NSApp nextEventMatchingMask: NSAnyEventMask untilDate: nil
-            inMode: NSDefaultRunLoopMode dequeue: NO ];
```

Comment:
it seems like this hack doesn't work anymore. I moved part of it to `didupdate` event